cgp/cartesian_graph.py

```diff
@@ -372,15 +398,25 @@ def to_sympy(self, simplify: Optional[bool] = True) -> List["sympy_expr.Expr"]:
 
         Returns
         ----------
-        List[sympy.core.expr.Expr]
-            List of SymPy expressions.
+        List[sympy.core.expr.Expr] or sympy.core.expr.Expr
+            List of SymPy expressions or single expression.
+
         """
+
+        def possibly_unpack(
```

Comment:
for my taste the code is more complicated with this helper function, than with two `if len(sympy_exprs) == 1:` below, but I guess it is a matter of taste... :) 
(also out of topic here, but do you think the option `simplify = False` is actually ever used in practise?)

Reply:
I agree with @HenrikMettler

Reply:
i've packed this into a function to avoid a comment, hoping the function name would show intention. ideas for other names?

`simplify=False` can be quite helpful, but it's mainly for debugging, not production i think.

Reply:
I would find a comment without a function easier to understand, to be honest.